tests/share/test_sources.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from furl import furl
+import pendulum
+import requests
+from requests.exceptions import Timeout
+from sickle import Sickle
+
+from share.models import SourceConfig
+
+
+@pytest.mark.django_db
+class TestSources:
```

Comment:
Yeah I was thinking that too. Especially since they could fail and have nothing to do with the PR. They should be run fairly frequently though?